Source/OxyPlot.SkiaSharp/SkiaRenderContext.cs

```diff
@@ -0,0 +1,762 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="SkiaRenderContext.cs" company="OxyPlot">
+//   Copyright (c) 2020 OxyPlot contributors
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot.SkiaSharp
+{
+    using global::SkiaSharp;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Implements <see cref="IRenderContext" /> based on SkiaSharp.
+    /// </summary>
+    public class SkiaRenderContext : IRenderContext, IDisposable
+    {
+        private SKPaint paint = new SKPaint();
+        private SKPath path = new SKPath();
+        private float dpiScale = 1;
+
+        /// <inheritdoc/>
+        public bool RendersToScreen { get; set; } = true;
+
+        /// <summary>
+        /// Gets or sets the <see cref="SKCanvas"/> the <see cref="SkiaRenderContext"/> renders to. This must be set before any draw calls.
+        /// </summary>
+        public SKCanvas SkCanvas { get; set; }
+
+        /// <inheritdoc/>
+        public void CleanUp()
+        {
+        }
+
+        /// <inheritdoc />
+        public void Dispose()
+        {
+            this.paint?.Dispose();
+            this.path?.Dispose();
+            this.paint = null;
+            this.path = null;
+        }
+
+        /// <inheritdoc/>
+        public void DrawEllipse(OxyRect extents, OxyColor fill, OxyColor stroke, double thickness, EdgeRenderingMode edgeRenderingMode)
+        {
+            if (!fill.IsVisible() && !(stroke.IsVisible() || thickness == 0))
```

Comment:
Should the thickness check be `<= 0`? Negative numbers don't make sense, but the later checks explicitly exclude them. Such checks also appear in e.g. `GraphicsRenderContext` (though that looks like a bit of mess in this regard).